modules/nw-ingress-sharding-route-configuration.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-in4ress-cluster-traffic-ingress-controller.adoc
+// * networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-route-configuration_{context}"]
+= Creating a route for Ingress Controller sharding
+
+A route allows you to host your application at a URL. In this case, the host name is not set and instead, the route uses a subdomain. When you specify a subdomain, you automatically use the domain of the Ingress Controller that exposes the route. For situations where a route is exposed by multiple Ingress Controllers, the route is hosted at multiple URLs.
+
+The following procedure describes how to create a route for Ingress Controller sharding, using the `hello-openshift` application as an example.
+
+Ingress Controller sharding is useful when balancing incoming traffic load among
+a set of Ingress Controllers and when isolating traffic to a specific Ingress
+Controller. For example, company A goes to one Ingress Controller and company B
+to another.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in as a project administrator.
+* You have a web application that exposes a port and an HTTP or TLS endpoint listening for traffic on the port.
+* You have configured the Ingress Controller for sharding.
+
+.Procedure
+
+. Create a project called `hello-openshift` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project hello-openshift
+----
+
+. Create a pod in the project by running the following command:
++
+[source,terminal]
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/hello-openshift/hello-pod.json
+----
+
+. Create a service called `hello-openshift` by running the following command:
++
+[source,terminal]
+----
+$ oc expose pod/hello-openshift
+----
+
+. Create a route defintion called `hello-openshift-route.yaml`:
```

Comment:
```suggestion
. Create a route definition called `hello-openshift-route.yaml`:
```

Reply:
@jldohmann  According to ISG, 'host name' is OK if updating older content where it already exists. So you can ignore my suggestions if this is the case. :-)

Reply:
this is new content, so I'll add your suggestion thanks for catching that!